packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/Examples.2/RowSelection.jsx

```diff
@@ -146,6 +281,12 @@ const RowSelection = () => {
           disableSelectableRows();
         }}
         onColumnSelect={onColumnSelect}
+        onEnableRowSelection={() => {
```

Comment:
You would not want to use an anonymous function.  We would not want that pattern replicated.

Reply:
Changed